### DISCUSSION TITLE
Setup Installation Failed when using 10.0.22621.0v signtool.exe to sign a Setup

### DISCUSSION THREAD

**Discussion by Naganathan** [oleg-shilo/wixsharp#1632] [General]
Hi,

I am using latest signtool 10.0.22621.0v to sign the setup in jenkins command (provided in below) instead of wixsharp source. But the signed setup.exe is not able to install. 

`"C:\Program Files (x86)\Windows Kits\10\bin\10.0.22621.0\x64\signtool.exe" sign /v /s MY /n "Digital Code Signing" /tr http://timestamp.digicert.com /td sha256 /fd sha256 /a "D:\Setup\MySetup.exe" `

Everthough, I am signing from WixSharp source still same issue I am facing. 

**setup installation error**

![image](https://github.com/user-attachments/assets/9c446881-6855-4ed0-ae45-51fda23d9a3d)

**Log file error:**

![image](https://github.com/user-attachments/assets/6a7f4762-852e-4a36-af0f-1c335f12f1b1)

Could you please help me on this? what I am doing wrong here.

**Comment by oleg-shilo** [comment]
Sorry for stating the obvious, but it seems to be a signing but not an MSI building problem.
Still, adding "help needed"  label in case if someone ca help you.

**Comment by saiprabhur** [comment]
found the solution
https://wixtoolset.org/docs/v3/overview/insignia/
i have followed the steps provided in the url using insignia the problem resolved.
